src/app/shared/components/NonInteractiveField/index.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Player, TeamSheet } from '@prisma/client'
+
+import NonInteractivePlayerIcon from './NonInteractivePlayerIcon'
+
+const teamSheetLayoutData = {
+  field: [
+    { teamSheetLayoutId: '1', style: { top: '70px', left: '140px' } },
+    { teamSheetLayoutId: '2', style: { top: '70px', left: '360px' } },
+    { teamSheetLayoutId: '3', style: { top: '70px', left: '580px' } },
+    { teamSheetLayoutId: '4', style: { top: '150px', left: '250px' } },
+    { teamSheetLayoutId: '5', style: { top: '150px', left: '470px' } },
+    { teamSheetLayoutId: '6', style: { top: '230px', left: '140px' } },
+    { teamSheetLayoutId: '7', style: { top: '230px', left: '360px' } },
+    { teamSheetLayoutId: '8', style: { top: '230px', left: '580px' } },
+    { teamSheetLayoutId: '9', style: { top: '330px', left: '140px' } },
+    { teamSheetLayoutId: '10', style: { top: '330px', left: '360px' } },
+    { teamSheetLayoutId: '11', style: { top: '430px', left: '140px' } },
+    { teamSheetLayoutId: '12', style: { top: '390px', left: '250px' } },
+    { teamSheetLayoutId: '13', style: { top: '390px', left: '470px' } },
+    { teamSheetLayoutId: '14', style: { top: '440px', left: '580px' } },
+    { teamSheetLayoutId: '15', style: { top: '490px', left: '360px' } },
+  ],
+}
+
+interface NonInteractiveFieldProps {
+  players: Player[],
+  teamSheet: TeamSheet,
+}
+
+const NonInteractiveField = (props: NonInteractiveFieldProps) => {
+  const { players, teamSheet } = props
+
+  const data = teamSheet.data as Partial<{ [key: string]: number }>
+
+  return (
+    <div style={{ display: 'flex' }}>
+      <img
+        height="630"
+        src={`${process.env.NEXT_PUBLIC_VERCEL_URL}/rugby_field.svg`}
```

Comment:
` ⨯ Error: failed to pipe response
    at pipeToNodeResponse (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/pipe-readable.js:106:15)
    at async sendResponse (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/send-response.js:41:17)
    at async doRender (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/base-server.js:1294:21)
    at async cacheEntry.responseCache.get.incrementalCache.incrementalCache (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/base-server.js:1500:28)
    at async DevServer.renderToResponseWithComponentsImpl (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/base-server.js:1408:28)
    at async DevServer.renderPageComponent (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/base-server.js:1760:24)
    at async DevServer.renderToResponseImpl (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/base-server.js:1798:32)
    at async DevServer.pipeImpl (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/base-server.js:851:25)
    at async NextNodeServer.handleCatchallRenderRequest (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/next-server.js:266:17)
    at async DevServer.handleRequestImpl (/Users/david.carr-gregg/code/dcg/teamsheet/node_modules/next/dist/server/base-server.js:748:17) {
  [cause]: Error: Image source must be an absolute URL: undefined/rugby_field.svg`

Reply:
probs caused by a missing env var
```
NEXT_PUBLIC_VERCEL_URL=http://localhost:3000
```

Reply:
🤦